Assets/Scripts/Managers/GameStateManager.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameStateManager : MonoBehaviour
+{
+    public static GameStateManager Instance;
```

Comment:
Can you make this read-only? So something like {get; private set;}

Reply:
ah right, got it